modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/timeline/Timeline.js

```diff
@@ -49,15 +49,15 @@ TimelineEntry.propTypes = {
 
 TimelineEntry.defaultProps = {};
 
-function Timeline({dataSetContext, items}) {
+function Timeline({frontendDataSetContext, items}) {
 	return (
 		<ClayList className={classNames('mb-0', 'timeline')}>
 			{items.map((item, i) => (
 				<TimelineEntry
 					key={i}
 					{...item}
 					borderBottom={i !== items.length - 1}
-					dataSetContext={dataSetContext}
+					frontendDataSetContext={frontendDataSetContext}
```

Comment:
This seems a bit weird 🤔 
The view variable is already wrapped in the context provider in `FrontendDataSet.js` line 744 🤷

Reply:
Yeah, I very much agree. This _might_ make sense if the component is exported and used from some other module, but it doesn't look like that's the case. This is on the parts of code that don't currently work, views other than table, so it's impossible to test. We can refactor these as part of tasks to enable those views.